### DISCUSSION TITLE
Tagi oraz etykiety aukcji w danych zwracanych przez /sale/offers/{offer-id}

### DISCUSSION THREAD

**Discussion by javaguru-pl** [allegro/allegro-api#4605] [Suggestions - rejected]
:bulb: **Sugestia**
Czy jest możliwość aby w/w metoda zwracała także listę **tagów** oraz **etykiet** danej oferty?
Teoretycznie wg dokumentacji ta metoda zwraca **WSZYSTKIE** pola danej oferty. Wg mnie tagi i etykiety oferty też są polami/atrybutami danej oferty.

**Comment by PrzemyslawLukanowski** [comment]
Zapoznaj się proszę z [naszym newsem](https://github.com/allegro/allegro-api/issues/4337). Zgodnie z nim przenoszę ten wątek do zakładki [Discussions](https://github.com/allegro/allegro-api/discussions), gdzie również inni użytkownicy będą mogli głosować na tę funkcjonalność, co pozwoli określić jej priorytet. Jeśli zdecydujemy się na wdrożenie propozycji - poinformujemy o statusie prac w wątku. Jednocześnie prosiłbym o informację, jakie dokładnie etykiety w API masz na myśli oraz uzasadnienie - jaka jest potrzeba dla tej zmiany.

**Comment by javaguru-pl** [comment]
Jeżeli chodzi o etykiety to przesyłam wyjaśnienie naszego klienta:

> Etykiety nadaje się w zakładce "moje oferty" (oznaczając aukcje i dalej "etykiety" -> "oznacz oferty"). Etykiety nie są widoczne dla kupującego. Aukcje oznaczone etykietami: 10086817982, 10487249873 - pewnie tego nie da się pokazać.

Uzasadnienie tej zmiany jest takie iż obecnie aby pobrać dane o tagach - to dla każdej aukcji musimy wywołać dodatkową metodę /sale/offers/{offerId}/tags. W sytuacji gdy przetwarzamy miliony zdarzeń i ofert dziennie - spowoduje to iż nasz limit wywołań RPS będzie musiał wzrosnąć o dobre kilkadziesiąt procent (zamiast jednego requestu do /sale/offers/{offer-id} potrzebujemy wykonać jeszcze dodatkowy request do /sale/offers/{offerId}/tags.

Tak więc główny powód takiej zmiany jest związany z wydajnością oraz limitami nałożonymi na wywołania API.

**Reply by PrzemyslawLukanowski** [reply]
Dziękuję za uzasadnienie. Co do samych etykiet - na tę chwilę nie planujemy dodawać tej funkcjonalności w API.

**Comment by MaciejFrackowiak** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy sugerowanej zmiany.
